internal/transformations/base64decode.go

```diff
@@ -29,11 +32,11 @@ var base64DecMap = []byte{
 // would be possible to use the standard library only relying on undocumented behaviors of the decoder.
 // For more context, see https://github.com/corazawaf/coraza/pull/940
 func base64decode(data string) (string, bool, error) {
-	res := doBase64decode(data)
+	res := doBase64decode(data, false)
 	return res, true, nil
 }
 
-func doBase64decode(src string) string {
+func doBase64decode(src string, ext bool) string {
```

Comment:
name is meaningful. I would use typed constants for this or at least add a comment on what ext is.

Reply:
I have added a comment regarding the flag.